frontend/src/Pages/Login Page/Login.module.css

```diff
@@ -0,0 +1,107 @@
+
+.login-page{
+    height: 100vh;
+    margin: 0;
+    overflow: auto;
+    display: flex;
+    flex-direction: column;
+}
+
+.login-body-container{
+    flex: 1;
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    text-align: center;
+}
+
+.login-body{
+    display:block;
+    padding:1rem;
+}
+
+.login-header{
+    background-color: #1B7D4F;
+    display: flex;
+    justify-content: space-between;
+    padding: 16px;
```

Comment:
Awesome use of responsive CSS! Great Work :)
Small Change : 
For elements like padding, gap, width, height, etc., change the px to rem or em throughout this CSS file

Reasoning: 
rem and em units allow more responsiveness as relative measurements are flexible to its parent element. For example, 16px on a screen size like a laptop is completely different from 16px screen size on a mobile device and designs would begin to get squished.